front/src/Settings/index.js

```diff
@@ -31,15 +31,15 @@ const COMMAND_VALUES = {
 };
 
 const Settings = props => {
-  const { turtles } = props;
+  const { turtles, token } = props;
```

Comment:
Please add "token" to the props in the documentation

Reply:
done